Extension/src/nativeStrings.json

```diff
@@ -13,7 +13,6 @@
     "could_not_find_compile_commands": "\"{0}\" could not be found. 'includePath' from c_cpp_properties.json in folder '{1}' will be used instead.",
     "file_not_found_in_path": "\"{0}\" not found in \"{1}\". 'includePath' from c_cpp_properties.json in folder '{2}' will be used for this file instead.",
     "database_reset": "The IntelliSense database was successfully reset.",
-    "global_scope": "",
```

Comment:
Would it be worth keeping this around to avoid breaking strings for any users who use the master branch and have the 0.28.1 binaries still?